### ISSUE TITLE
Fetch a json file from FS

### ISSUE THREAD

**Issue by felipebmendes** [jupyterlab/jupyterlab#6144]
I'm working on the development of a jupyterlab extension and I'm trying to fetch a json file from my filesystem using the `makeRequest` function from the `@jupyterlab/services` package but I'm always getting a 404 NOT FOUND error.

Should it be working using only the `makeRequest` function or do I have to develop another extension to manage the access to filesystem working as an API on server side?

This is my activate function:

```
function activate(app: JupyterLab) {
  console.log('JupyterLab extension is activated!');
  let serverSettings: ServerConnection.ISettings = ServerConnection.makeSettings();
  console.log(serverSettings);
  let url: string = 'config.json';
  let fullURL = URLExt.join(serverSettings.baseUrl, url);
  ServerConnection.makeRequest(fullURL, {
      method: "GET"
  }, serverSettings).then(response => {
      if (response.status !== 200) {
          throw new ServerConnection.ResponseError(response)
      }
      console.log(response.json());
  });
}
```

Thanks in advance.

**Comment by bollwyvl** [comment]
Perhaps try putting 'files' between baseUrl and url. This _should_ return
the file directly, without wrapping it in a content model, assuming it's in
the root of the notebook_dir the server started with.

**Comment by jasongrout** [comment]
Yes, either use the `/files` prefix to the files url, or use the contents manager api to retrieve the file.
